packages/esm-patient-registration-app/src/index.ts

```diff
@@ -12,17 +12,6 @@ import { setupOffline } from './offline';
 
 const importTranslation = require.context('../translations', false, /.json$/, 'lazy');
 
-const backendDependencies = {
```

Comment:
Out of curiosity: Are we not using this anymore?

Reply:
No, we are not using it anymore. RIght @ibacher?

https://github.com/openmrs/openmrs-esm-patient-management/pull/196#discussion_r893563319

Reply:
@vasharma05 We dropped support for `frontendDependencies`, but `backendDependencies` is still used and important!

Reply:
Thanks @ibacher!
I have restored the backend dependencies.